src/Handler.php

```diff
@@ -40,7 +40,7 @@ public function __construct(array $middleware, callable $default)
      *
      * @return ResponseInterface
      */
```

Comment:
Docblock can be removed now that return type is specified.

Reply:
The whole docblock or just the `@return` part?